camel/messages/base.py

```diff
@@ -167,37 +164,32 @@ def extract_text_and_code_prompts(
 
         return text_prompts, code_prompts
 
-    def to_openai_message(self, role: Optional[str] = None) -> OpenAIMessage:
+    def to_openai_message(self, role: str) -> OpenAIMessage:
```

Comment:
```suggestion
    def to_openai_message(self, role_name: str) -> OpenAIMessage:
```
memorizing the third role- parameter and distinguishing it from the others is hard to me

Reply:
What do you mean by the third? We have just two parameters here.

Reply:
I think calling `role` or `role_at_backend` is good.

Reply:
Renaming to role_at_backend.